ci/travis/install-dependencies.sh

```diff
@@ -337,13 +337,13 @@ install_dependencies() {
     # If CI has deemed that a different version of Tensorflow or Torch
     # should be installed, then upgrade/downgrade to that specific version.
     if [ -n "${TORCH_VERSION-}" ] || [ -n "${TFP_VERSION-}" ] || [ -n "${TF_VERSION-}" ]; then
-      case "${TORCH_VERSION-1.7}" in
-        1.7) TORCHVISION_VERSION=0.8.1;;
+      case "${TORCH_VERSION-1.8}" in
```

Comment:
If the tests pass then this is fine, otherwise I think we would have to still keep the 1.7 case if RLlib doesn't work with 1.8. The Torch installs in this file actually are not used for Tune/SGD/Doc testing, only for RLlib and Serve.